src/sentry/tagstore/snuba/backend.py

```diff
@@ -309,11 +310,19 @@ def __get_tag_keys_for_projects(
             # Cause there's rounding to create this cache suffix, we want to update the query end so results match
             end = snuba.quantize_time(end, key_hash)
             cache_key += f":{duration}@{end.isoformat()}"
-            result = cache.get(cache_key, None)
-            if result is not None:
-                metrics.incr("testing.tagstore.cache_tag_key.hit")
-            else:
-                metrics.incr("testing.tagstore.cache_tag_key.miss")
+
+            with sentry_sdk.start_span(op="cache.get", name="tagstore.cache") as span:
```

Comment:
Just a note, if we instrument more, we'll probably have to standardize the span description `<cache namespace>.cache`. Not sure if namespace is the right word lol, but do you get what i mean?

Reply:
I made the namespace more specific, should be better now!